pyproject.toml

```diff
@@ -93,3 +93,31 @@ convention = "numpy"
 [tool.pytest.ini_options]
 # Silence cotengra warning
 filterwarnings = ["ignore:Couldn't import `kahypar`"]
+
+[tool.mypy]
+disable_error_code = [
+  "type-arg",
+  "unused-ignore", # Necessary for pyright
```

Comment:
Can you explain why this line is necessary? I don't see any error from either `mypy` or `pyright` when it is removed.

Reply:
This can be removed if we use `pyright: ignore` instead of `type: ignore` when only `pyright` complains.

Reply:
Fixed in 74ce7925275807dd18f21b301f3f659b6d5dc0f1.